src/narrow-screen/components/FilterSlideUpPanel.js

```diff
@@ -138,7 +138,7 @@ const SectionTitle = styled.i`
 `;
 
 const SaveButton = styled(Button)`
-  background-color: #F3A35C;
+  background-color: #F9D5B4 !important;
```

Comment:
idk why the asana task says this hex value, but it's apparently not the same as what's in figma... Can we double check on that?
<img width="695" alt="Screen Shot 2020-09-02 at 5 01 20 PM" src="https://user-images.githubusercontent.com/24697473/91961581-efc9f700-ed3d-11ea-98a9-3dbe91438384.png">

Reply:
hmm this was something @cchyung told me to change, should I change it back?

Reply:
zach and i went over this personally - we should use the hex code from asana